package com.byt3social.analisedocumental.exceptions;

public class DadoNotFoundException extends RuntimeException {
    public DadoNotFoundException() {
        super("Dado n√£o encontrado no processo informado");
    }
}
